registry/format.md

```diff
@@ -21,9 +21,9 @@ Please raise a [Pull-Request](https://github.com/OAI/OpenAPI-Specification/pulls
 
 ## Values
 
-For the purpose of [JSON Schema validation](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-7.1), each format should specify the set of instance types (primitive JSON types) for which it applies. In this registry, these types are shown in the "Instance Type" column.
+For the purpose of [JSON Schema validation](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-7.1), each format should specify the set of JSON data types for which it applies. In this registry, these types are shown in the "JSON data type" column.
 
-|Value|Description|Instance Type|Source|Deprecated|
+|Value|Description|JSON data Type|Source|Deprecated|
```

Comment:
```suggestion
|Value|Description|JSON Data Type|Source|Deprecated|
```
Sorry for being nitpicky

Reply:
Why do we need a capital D in your view?

Reply:
Because its a table header, and we usually Uppercase Everything in a header cell.

Which @handrews did in #4045.